filament/backend/src/opengl/platforms/PlatformCocoaTouchGL.mm

```diff
@@ -108,11 +108,17 @@
 }
 
 Platform::SwapChain* PlatformCocoaTouchGL::createSwapChain(uint32_t width, uint32_t height, uint64_t& flags) noexcept {
-    // TODO: implement headless SwapChain
-    return nullptr;
+    CAEAGLLayer* glLayer = [CAEAGLLayer layer];
+    glLayer.frame = CGRectMake(0, 0, width, height);
+    return (SwapChain*) CFBridgingRetain(glLayer);
 }
 
 void PlatformCocoaTouchGL::destroySwapChain(Platform::SwapChain* swapChain) noexcept {
+    CAEAGLLayer* glLayer = (CAEAGLLayer*) CFBridgingRelease(swapChain);
```

Comment:
The problem with this approach is that we'll always call `release` on `swapChain`, even if it's not a headless one (and we didn't instantiate it ourselves).

In the [desktop implementation](https://github.com/google/filament/blob/dbf51c1c7de27407a25eecd31a78a08ff3341aed/filament/backend/src/opengl/platforms/PlatformCocoaGL.mm#L192-L193), the headless SwapChains get added to a `mHeadlessSwapChains` vector (which maintains their reference count) and then get removed (freeing their reference count) inside `destroySwapChain`.

 Come to think of it, I like this style better than calling `CFBridgingRetain`/`CFBridgingRelease`.